### PR TITLE
upgrade web3j dependency and regenerate wrappers

### PR DESCRIPTION
**Pull request by thewhitewizard**

### CODE REVIEW

**Review by jbern0rd — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+```
+git clone https://github.com/iExecBlockchainComputing/PoCo-dev/
+cd PoCo-dev
+git checkout x.y.z
+npm i
+./node_modules/.bin/truffle compile
+```
+
+```
+web3j generate truffle --truffle-json ~/iexecdev/PoCo-dev/build/contracts/App.json -o ~/iexecdev/iexec-commons-poco/src/main/java/ -p com.iexec.commons.poco.contract.generated
```
Comment (jeremyjams) [inline README.md:28]: Use script for main "easy" block?
```
./generateContractWrappers \
  App \
  AppRegistry \
  [...]
```

Comment (thewhitewizard) [reply README.md:27]: it's a good idea, I can make a small bash script

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+
+#Put here the src/main/java/ directory of commons-poco project
+COMMONS_POCO_WRAPPER_DIRECTORY=${HOME}/iexecdev/iexec-commons-poco/src/main/java/
+```
+
+Clean `IexecLibOrders_v5.OrderOperationEnum` references from `IexecLibOrders_v5.json` with your favorite editor(Only in abi node).
+
+You can now run the script
+``` shell
+./generateContractWrappers App AppRegistry AppRegistry Dataset DatasetRegistry Ownable Workerpool WorkerpoolRegistry IexecInterfaceTokenABILegacy IexecLibOrders_v5
```
Comment (jbern0rd) [inline README.md:38]:
```suggestion
./generateContractWrappers App AppRegistry Dataset DatasetRegistry Ownable Workerpool WorkerpoolRegistry IexecInterfaceTokenABILegacy IexecLibOrders_v5
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,41 @@
...
+```
+
+After this execution and if no error has occurred, you must rename `IexecInterfaceTokenABILegacy.java` to `IexecHubContract.java`
```
Comment (jbern0rd) [inline README.md:41]: Should we add a mention not to forget to update class name within the file ?

Comment (thewhitewizard) [reply README.md:41]: you're right, I'll add a precision

File: README.md (hunk 2 above)
Comment (thewhitewizard) [reply README.md:38]: well noted

**Review by jbern0rd — APPROVED** [review]

**Pull request merged**